packages/@n8n/config/src/index.ts

```diff
@@ -0,0 +1,8 @@
+import { Config, Nested } from './decorators';
+import { DatabaseConfig } from './configs/database';
+
+@Config
+export class GlobalConfig {
+	@Nested
+	database: DatabaseConfig;
+}
```

Comment:
We might consider utility methods, e.g. `restoreDefaults()` for tests.

Reply:
to restore defaults for tests, we can just create a new instance of the class.